lib/aullar/styling.moon

```diff
@@ -4,6 +4,7 @@
 {:fill} = require 'ffi'
 {:define_class} = require 'aullar.util'
 GapBuffer = require 'aullar.gap_buffer'
+Markers = require 'aullar.markers'
```

Comment:
I'm missing things from the implementation here - `set` and `insert` does not modify the sub style markers at all, which they reasonably should (since the build is green, specs are probably a good idea for these cases).